NicolaeChristof/Rise
# Feature season templates

I created a template for each of the four seasons. For each template I created a new post processing profile and assigned a new leaf to the branch UI. I also modified the template of each season to give a rough estimate of what that season could look like (for example spring has more checkpoints and less enemies).

To test:
Please load each scene template and check that
- Each camera is using the appropriate post processing profile (they are all pretty distinct from one another with winter looking the coldest and summer looking the hottest)
- Each template uses the appropriate leaf UI element
